shell/platform/darwin/ios/framework/Source/SemanticsObject.mm

```diff
@@ -258,14 +257,15 @@ @implementation FlutterCustomAccessibilityAction {
 @end
 
 @interface SemanticsObject ()
+@property(nonatomic, retain) SemanticsObjectContainer* container;
+
 /** Should only be called in conjunction with setting child/parent relationship. */
-- (void)privateSetParent:(SemanticsObject*)parent;
+@property(nonatomic, assign, readwrite) SemanticsObject* parent;
```

Comment:
Swap `privateSetParent:` to a `readwrite` property.

Reply:
nit: `readwrite` is already the default

Reply:
this will change to weak in arc right?

Reply:
Yes, I will make it weak in the ARC migration PR.

I was making the `readwrite` explicit since it's now overriding a `readonly` in the header:
```objc
@property(nonatomic, assign, readonly) SemanticsObject* parent;
```